app/src/main/res/values/strings.xml

```diff
@@ -644,4 +643,6 @@
 	<!-- Text for button which does not discard changes and keeps editing. -->
 	<string name="discard_host_cancel_button">Keep editing</string>
 
+	<string name="pref_alwaysvisible_title">Keyboard always visible</string>
```

Comment:
nit: I'd rather just have this say "Special keys always visible." not sure we even need the description. with "keyboard" it's a little ambiguous whether you mean the soft keyboard or special keys.